src/components/atoms/SortDropDown/SortDropDown.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import {
+  Dropdown as ReactBootstrapDropdown,
+  DropdownButton,
+} from "react-bootstrap";
+
+import { SortingOptions } from "utils/venue";
+
+import "./SortDropDown.scss";
+
+interface SpaceSortDropDownProps {
+  onClick?: (value: SortingOptions) => void;
```

Comment:
Should this be optional?

I can't see any point in using this dropdown without a callback provided.

Reply:
I was thinking about this, then I went to fix something else, I forgot I guess

Reply:
removed the `?`  @ b7425bd8196ff5942d4c257358cf401de6df8750